worlds/alttp/Bosses.py

```diff
@@ -166,10 +182,10 @@ def GanonDefeatRule(state, player: int) -> bool:
     ]
 
 
-def place_plando_bosses(bosses: List[str], world, player: int) -> Tuple[List[str], List[Tuple[str, str]]]:
+def place_plando_bosses(bosses: List[str], multiworld, world, player: int) -> Tuple[List[str], List[Tuple[str, str]]]:
     # Most to least restrictive order
     boss_locations = boss_location_table.copy()
-    world.random.shuffle(boss_locations)
+    multiworld.random.shuffle(boss_locations)
```

Comment:
we only use multiworld here for random. see the other comment about random

Reply:
I am keeping this intentionally like this for now.